v2/kafka-to-bigquery/src/main/java/com/google/cloud/teleport/v2/options/KafkaToBigQueryFlexOptions.java

```diff
@@ -361,4 +445,32 @@ public interface KafkaToBigQueryFlexOptions
   String getOutputDeadletterTable();
 
   void setOutputDeadletterTable(String outputDeadletterTable);
+
+  @TemplateParameter.Boolean(
```

Comment:
left over code?

Reply:
Oh! I added that initially for staging the template, let me remove that